src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package Duke;
+
+import Duke.Exception.DataCorruptedException;
+import Duke.Exception.EmptyArgumentException;
+import Duke.Exception.WrongArgumentException;
+import Duke.Tasks.TaskType;
+import Duke.Tasks.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.Scanner;
+
+
+public class Duke {
+    private static TaskList list = new TaskList();
+    private static boolean toSave = true;
+
+    private static final String SEPARATOR = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        final String DUKE_LOGO =  " ____        _\n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        final String CONV_START = DUKE_LOGO + "Hello! I'm Duke";
+        final String CONV_END = "Bye. Hope to see you again soon!";
+
+
+        printOutput(CONV_START);
+
+        boolean initialiseSuccessful = true;
+
+        try {
+            list.initialiseTaskFromFile();
+        } catch (FileNotFoundException e) {
+            printError("File not found. Make sure the data file is located in './data/data.md'"
+            + "\nSaving services disabled.");
+            toSave = false;
+        } catch (DataCorruptedException e) {
+            printError("File data corrupted. Please check data file again or delete entire content."
+            + "\nExiting program.");
+            initialiseSuccessful = false;
+        }
+
+        Scanner in = new Scanner(System.in);
+
+        if (initialiseSuccessful) {
+            if (list.getTaskListSize() != 0) {
+                printOutput(list.getCompleteList());
+            } else if (toSave) {
+                try {
+                    list.addMarkdownHeader();
+                } catch (IOException e) {
+                    printError("Something wrong happen, error data: " + e.getMessage());
+                }
+            }
+        }
+
+        while (initialiseSuccessful) {
+            System.out.println("What can I do for you?");
+            String lineInput = in.nextLine();
+            String[] inputSplitted = lineInput.split(" ",2);
```

Comment:
In such cases, you may also want to consider relocating your delimiter to a variable, so as to eliminate magic literals.